plugins/modules/database/influxdb/influxdb_database.py

```diff
@@ -16,7 +16,7 @@
 author: "Kamil Szczygiel (@kamsz)"
 requirements:
     - "python >= 2.6"
-    - "influxdb >= 0.9 & <= 1.2.4"
+    - "influxdb >= 0.9"
```

Comment:
Did you check whether other influxdb_* modules had the same restriction?

Reply:
All other influxdb modules only have the `"influxdb >= 0.9"` requirement